CMakeLists.txt

```diff
@@ -259,7 +264,9 @@ else()
   list(APPEND _ufs_defs_private FRONT_FV3=fv3gfs_cap_mod)
   add_dependencies(ufs fv3atm)
 endif()
-
+if(NEMS_DRIVER)
```

Comment:
We'd like to have NEMS_DRIVER as default, and use JEDI_DRIVER if the model needs to be run within JEDI.

Reply:
I'll make the changes to the nomenclature.